go.mod

```diff
@@ -13,13 +13,13 @@ require (
 	github.com/google/uuid v1.6.0
 	github.com/hashicorp/go-plugin v1.6.2
 	github.com/jackc/pgx/v4 v4.18.3
-	github.com/jpillora/backoff v1.0.0
 	github.com/lib/pq v1.10.9
-	github.com/pelletier/go-toml/v2 v2.2.0
-	github.com/prometheus/client_golang v1.17.0
+	github.com/pelletier/go-toml/v2 v2.2.2
+	github.com/prometheus/client_golang v1.20.5
 	github.com/smartcontractkit/chainlink-common v0.4.1-0.20241223143929-db7919d60550
+	github.com/smartcontractkit/chainlink-framework/multinode v0.0.0-20250114145318-c816a400dcb3
```

Comment:
this should be pointed to a merged commit on the `main` branch of `chainlink-framework` otherwise it's possible that the commit will get GC'd in the future

Reply:
Sounds good, will merge [Add NewHeadsPollInterval](https://github.com/smartcontractkit/chainlink-framework/pull/6) first.